Compiling with java 11 errors

## What did you run exactly?
Compile the sources using java 11

## Actual behavior
Errors with the following compilation problem:
```
/Users/adietish/Documents/jboss-workspaces/intellij-openshift-connector/src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationTreeModel.java:167: error: incompatible types: inference variable T has incompatible equality constraints Object,CAP#1
          return Collections.list(((MutableTreeNode)o).children());
                                 ^
  where T is a type-variable:
    T extends Object declared in method <T>list(Enumeration<T>)
  where CAP#1 is a fresh type-variable:
    CAP#1 extends TreeNode from capture of ? extends TreeNode
```

--- a/src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationTreeModel.java
+++ b/src/main/java/org/jboss/tools/intellij/openshift/tree/application/ApplicationTreeModel.java
@@ -34,6 +34,7 @@ import javax.swing.tree.TreePath;
 import java.io.File;
 import java.io.IOException;
 import java.util.Collections;
+import java.util.Enumeration;
 import java.util.HashMap;
 import java.util.List;
 import java.util.Map;
@@ -167,7 +168,7 @@ public class ApplicationTreeModel extends BaseTreeModel<Object> implements Confi
             if (!node.isLoaded()) {
                 node.load();
             }
-            return Collections.list(((MutableTreeNode)o).children());
+            return Collections.list((Enumeration) ((MutableTreeNode)o).children());
 
         } else {
             return Collections.emptyList();

